package.json

```diff
@@ -11,7 +11,25 @@
     "acorn.js",
     "interpreter.js",
     "LICENSE",
-    "README.md"
+    "README.md",
+    "bin"
   ],
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "bin": "bin/run.js",
+  "resolutions": {
+    "eshost": "pcardune/eshost#f3fa7fa",
+    "test262-compiler": "pcardune/test262-compiler#af9df33"
```

Comment:
I've opened a pull request with the change I made to this project. Hopefully it will get looked at / merged soon: https://github.com/bterlson/test262-compiler/pull/1